apps/src/StudioApp.js

```diff
@@ -1834,23 +1836,30 @@ StudioApp.prototype.fixViewportForSmallScreens_ = function(viewport, config) {
     if (this.noPadding && deviceWidth < MAX_PHONE_WIDTH) {
       desiredWidth = Math.min(desiredWidth, mobileNoPaddingShareWidth);
     }
-    minWidth = mobileNoPaddingShareWidth;
+    var minWidth = mobileNoPaddingShareWidth;
+    width = Math.max(minWidth, desiredWidth);
+    scale = deviceWidth / width;
   } else {
-    // assume we are in landscape mode, so width is the longer of the two
-    deviceWidth = desiredWidth = Math.max(screen.width, screen.height);
-    minWidth = MIN_WIDTH;
+    // We want the longer edge, the width in landscape, to get MIN_WIDTH.
+    let screenWidth = Math.max(screen.width, screen.height);
+
+    width = MIN_WIDTH;
+    scale = screenWidth / width;
   }
-  var width = Math.max(minWidth, desiredWidth);
-  var scale = deviceWidth / width;
 
+  // Setting `minimum-scale=scale` means that we are unable to shrink the
+  // entire playspace area down to fit on a portrait iPhone, even though
+  // it's technically not visible because of the RotateContainer on top.
+  // But setting `maximum-scale=scale` was preventing an Android from starting
+  // in the desired zoom level in landscape, so we removed that but left
+  // `minimum-scale=scale`.
```

Comment:
Adding some more detail for documentation purposes: We found that `initial-scale=scale` and `minimum-scale=scale` and **not** `maximum-scale=scale` were required to make Android render correctly on initial page load while allowing users to manually zoom/resize with two-finger gestures.
Full details here: https://developer.mozilla.org/en-US/docs/Mozilla/Mobile/Viewport_meta_tag